specs/WebRtcPortConfiguration.md

```diff
@@ -77,9 +80,47 @@ var environment = await CoreWebView2Environment.CreateAsync(
 OnCreateEnvironmentCompleted(environment);
 ```
 
+API Rules and Precedence
+
+1. Network Scope param in SetAllowedPortRange
+- A network component-specific scope (e.g. _WEB_RTC) always takes precedence over _ALL for that component in `SetAllowedPortRange`.
+- `_ALL` defines the port range restrictions for all components without specific overrides.
+- Passing `(0, 0)` for a network component scope removes its restriction.
+- If `_ALL` is set and a specific scope is reset, that component becomes unrestricted while `_ALL` still applies to others.
+
+| Network Scope State                        | Behaviour                                                                      |
+| ------------------------------------------ | ------------------------------------------------------------------------------ |
+| Only `_ALL` is set                         | `_ALL` applies port range restrictions to all network components               |
+| `_ALL` and `_WEB_RTC` are both set         | `_WEB_RTC` port range restrictions applies to WebRTC; `_ALL` applies to others |
```

Comment:
The other candidate interaction model would be that when "All" and a specific Component both have ranges set, the effective range at runtime is the intersection of the two ranges.  It's not immediately clear to me if this model would be more or less useful to developers.  Just checking, have both models been considered?